src/lib/utils/thorchain/index.ts

```diff
@@ -1,8 +1,15 @@
-import type { AccountId } from '@shapeshiftoss/caip'
-import { type AssetId, bchChainId, fromAccountId, fromAssetId } from '@shapeshiftoss/caip'
+import type { AccountId, ChainId } from '@shapeshiftoss/caip'
+import {
+  type AssetId,
```

Comment:
i can't help but always notice these lol

Reply:
blame the LSP